afl-fuzz.c

```diff
@@ -3454,9 +3464,22 @@ static void write_stats_file(double bitmap_cvg, double stability, double eps) {
              persistent_mode ? "persistent " : "", deferred_mode ? "deferred " : "",
              (qemu_mode || dumb_mode || no_forkserver || crash_mode ||
               persistent_mode || deferred_mode) ? "" : "default",
-             orig_cmdline);
+             orig_cmdline, slowest_exec_ms);
              /* ignore errors */
 
+  /* Get rss value from the children
+     We must have killed the forkserver process and called waitpid
+     before calling getrusage */
+  if (getrusage(RUSAGE_CHILDREN, &usage)){
+      WARNF("getrusage failed");
+  }
+  else if (usage.ru_maxrss == 0){
+    fprintf(f, "peak_rss_mb       : not available while afl is running\n");
+  }
+  else{
+    fprintf(f, "peak_rss_mb       : %zu\n", usage.ru_maxrss);
```

Comment:
I don't think this reports the correct value. See the [implementation in libFuzzer](https://cs.chromium.org/chromium/src/third_party/libFuzzer/src/FuzzerUtilPosix.cpp?l=144) for how to properly report rss.

Reply:
You're right! I forgot to convert in mb. I fixed it in #23.